deploy/ansible/roles/certbot/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+- name: Install Certbot
+  apt:
+    name: python3-certbot-nginx
+    state: present
+
+- name: Run Certbot
+  command: certbot -m infrastructure+api.bocoup.com@bocoup.com --agree-tos -n --domains {{api_fqdn}} --nginx
```

Comment:
```suggestion
  command: certbot -m {{letsencrypt_email}} --agree-tos -n --domains {{api_fqdn}} --nginx
```

Reply:
It looks like this suggestion got lost in the shuffle. Mind making the replacement?

Reply:
Sorry about that!  Thank you for the catch -- New commit with this change (and the newline): https://github.com/bocoup/people/pull/55/commits/288679cd883dd6b6666449b848416f83c00effcd